UPGRADE.md

```diff
@@ -36,6 +36,294 @@ starts new workers, which take over from old workers before those old workers
 are terminated. In this way, Kong will serve new requests via the new
 configuration, without dropping existing in-flight connections.
 
+## Upgrade to `0.11.x`
+
+Along with the usual database migrations shipped with our major releases, this
+particular release introduces a few changes in behavior and, most of all, the
+removal of the Serf dependency for cache invalidation between Kong nodes of a
+same cluster.
+
+This document will only highlight the breaking changes that you need to be
+aware of, and describe a recommended upgrade path. We recommend that you
+consult the full [0.11.0
+Changelog](https://github.com/Mashape/kong/blob/master/CHANGELOG.md) for a full
```

Comment:
"The full 0.11.0 Changelog for a more *thorough* list of changes and new features"